deploy/helm/sumologic/values.yaml

```diff
@@ -4022,9 +4026,9 @@ metadata:
           logs:
             receivers:
               - fluentforward
+              - otlp
```

Comment:
Let's create a separate pipeline for this as discussed elsewhere.

Also please rename this pipeline to `logs/containers` (and accordingly the other one can be called `logs/containers_otlp`)

Reply:
Isn't changing the name technically a breaking change?

Reply:
It is but not in this case because using OT distro gathering metadata is still in beta and we agreed that beta features can undergo breaking changes.

Reply:
Changed them all to `logs/fluent-bit/*` and the otlp one to `logs/otlp/containers`. With this change, I think the purpose of the fluent include filters is much clearer as well.